plsql-frontend/src/main/java/org/sonar/plugins/plsqlopen/api/PlSqlKeyword.java

```diff
@@ -35,6 +35,7 @@
     BEGIN("begin", true),
     BETWEEN("between", true),
     BY("by", true),
+    CACHE("cache", true),
```

Comment:
There's no reason to define these keywords as reserved word (passing "true" in the second parameter), since they are not on [this list](https://docs.oracle.com/database/121/SQLRF/ap_keywd001.htm). You can just define them like CACHE("cache") and move them to the end of the list with the other non-reserved keywords.